replace-B2G.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [[ ! -d "$B2G_PATH" ]]; then
+  B2G_PATH=../B2G
+fi
+
+make install
+make sync
+GAIA_DEV_PIXELS_PER_PX=2.25 make shallow
```

Comment:
The 2.25 is pretty device specific, is there a way to do this for a flame if we wanted to do so?